.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: '[name of the chart] <issue title>'
+labels: ''
+assignees: ''
+
+---
+
+<!-- Thanks for filing an issue! Before hitting the button, please answer these questions. It's helpful to search the existing GitHub issues first. It's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of
+
+Fill in as much of the template below as you can.  If you leave out information, we can't help you as well.
+
+Be ready for followup questions, and please respond in a timely manner. If we can't reproduce a bug or think a feature already exists, we might close your issue.  If we're wrong, PLEASE feel free to reopen it and explain why.
+-->
+
+**Bug description**
+
+A clear and concise description of what the bug is.
+
+**Version of Helm and Kubernetes**
+
+The versino of Helm and Kubernetes where the problem occurs.
+
+**Which chart?**
+
+The chart name and version.
+
+**What happened?**
+
+Described what happened, as detailed as possible and necessary.
+
+**What you expected to happen?**
+
+Describe what you expected to happen. Please, avoid saying "it works".
```

Comment:
```Please, avoid saying "it works".```

I would remove the comma, it sounds a bit better imo

Reply:
`Please avoid saying "I should just work"`

Reply:
Fixed in 5dc6a4f